README.md

```diff
@@ -0,0 +1,71 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+> Description the project.
```

Comment:
- [ ]  I would recommend you write a proper description for your website which will make your repo look professional. By doing so, you are also selling yourself to your future recruiters 😄.